docs/blog/2019-11-11-gazette-october/index.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Gatsby Gazette: The "What happened to October?" Edition'
+date: 2019-11-11
+author: Mike Allanson
+excerpt: "It's November already. But what happened to October? Read more to find out."
+tags:
+  - gazette
+---
+
+October means halloween! For your spooky reading pleasure the blog now has a dark mode. Dim the lights, pull up a ocmfy chair and we can begin.
+
+In September we posted our [Series A funding announcement](https://www.gatsbyjs.org/blog/2019-09-26-announcing-gatsby-15m-series-a-funding-round/), then headed to the UK for Gatsby Days London. Fear not, if you missed all the action you can [catch up by watching the videos](https://www.youtube.com/watch?v=UvO7h_KwgpE&list=PLCU2qJekvcN2f9rgS50xy7-4QBpgwid4G).
```

Comment:
removing domain only in docs or also in blog posts?

```suggestion
In September we posted our [Series A funding announcement](/blog/2019-09-26-announcing-gatsby-15m-series-a-funding-round/), then headed to the UK for Gatsby Days London. Fear not, if you missed all the action you can [catch up by watching the videos](https://www.youtube.com/watch?v=UvO7h_KwgpE&list=PLCU2qJekvcN2f9rgS50xy7-4QBpgwid4G).
```

Reply:
More relevant for docs, this is fine, I think. Good catch though!